JapaneseFilter/sections/specific.txt

```diff
@@ -4043,6 +4043,7 @@ bellemaison.jp##.inc-ad-info
 novel.prcm.jp##.side-ad-wrapper
 novel.prcm.jp##.footer__ad
 novel.prcm.jp##app-remote-ad-frame
+nekotoningen.blog.jp##.app-follow-banner
```

Comment:
Please move it to - https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt